### PR TITLE
EAD-Session-8

### PR DESCRIPTION
**Pull request by meta-manoj-sharma**

### CODE REVIEW

File: EAD-Session-8/StudentSystem/src/main/java/com/metacube/student/controller/HomeController.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
+package com.metacube.student.controller;
```
Comment (jbhavya) [inline EAD-Session-8/StudentSystem/src/main/java/com/metacube/student/controller/HomeController.java]: Add unit test.

File: EAD-Session-8/StudentSystem/src/main/java/com/metacube/student/model/Student.java [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+	private String firstName;
+	@NotBlank(message = "{lastNameError}")
+	@Pattern(regexp = "[a-z A-Z]+", message = "{lastNameError.OnlyAlphabates}")
+	private String lastName;
+	@NotBlank(message = "{fatherNameError}")
+	@Pattern(regexp = "[a-z A-Z]+", message = "{fatherNameError.OnlyAlphabates}")
+	private String fatherName;
+
+	@NotBlank(message = "{emailBlank}")
+	@Email(message="{emailPattern}")
```
Comment (jbhavya) [inline EAD-Session-8/StudentSystem/src/main/java/com/metacube/student/model/Student.java]: Email validations are not proper.

File: EAD-Session-8/StudentSystem/src/main/webapp/WEB-INF/pages/showStudent.jsp [hunk 3]
```diff
@@ -0,0 +1,37 @@
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1"
+	pageEncoding="ISO-8859-1"%>
+<%@page import="java.util.*"%>
+<%--Importing all the dependent classes--%>
+
+<%@page import="java.util.Iterator"%>
```
Comment (jbhavya) [inline EAD-Session-8/StudentSystem/src/main/webapp/WEB-INF/pages/showStudent.jsp]: Remove this import statement when you have imported the whole package above.

File: EAD-Session-8/StudentSystem/src/main/java/com/metacube/student/controller/HomeController.java (hunk 1 above)
Comment (meta-manoj-sharma) [reply EAD-Session-8/StudentSystem/src/main/java/com/metacube/student/controller/HomeController.java]: added

File: EAD-Session-8/StudentSystem/src/main/webapp/WEB-INF/pages/showStudent.jsp (hunk 3 above)
Comment (meta-manoj-sharma) [reply EAD-Session-8/StudentSystem/src/main/webapp/WEB-INF/pages/showStudent.jsp]: removed

File: EAD-Session-8/StudentSystem/src/main/java/com/metacube/student/model/Student.java (hunk 2 above)
Comment (meta-manoj-sharma) [reply EAD-Session-8/StudentSystem/src/main/java/com/metacube/student/model/Student.java]: updated logic

**Pull request merged**